src/mame/juku/juku.cpp

```diff
@@ -512,17 +657,23 @@ void juku_state::juku(machine_config &config)
 	m_sio[1]->rxrdy_handler().set("pic", FUNC(pic8259_device::ir0_w));
 	m_sio[1]->txrdy_handler().set("pic", FUNC(pic8259_device::ir1_w));
 
-	screen_device &screen(SCREEN(config, "screen", SCREEN_TYPE_RASTER));
-	screen.set_refresh_hz(60);
-	screen.set_vblank_time(ATTOSECONDS_IN_USEC(2500)); // not accurate
-	screen.set_size(320, 240);
-	screen.set_visarea(0, 319, 0, 239);
-	screen.set_screen_update(FUNC(juku_state::screen_update));
+	// Электроника МС 6105.4 (60 Hz)
+	SCREEN(config, m_screen, SCREEN_TYPE_RASTER);
+	m_screen->set_refresh_hz(60);
+	m_screen->set_vblank_time(ATTOSECONDS_IN_USEC(2500)); // not accurate
+	m_screen_x = 320, m_screen_y = 240;
```

Comment:
Don't assign variables here - maybe set it at `machine_reset` time and just use fixed values here.